### PR TITLE
[astute] Remove unusable message of successful RH licence checking.

### PR DESCRIPTION
**Pull request by adanin**
Not tested. Should works fine.

### CODE REVIEW

**Comment by mihgen** [comment]
1) did you run rspec?
2) did you try to run it?

File: lib/astute/redhat_checker.rb [hunk 1]
```diff
@@ -94,8 +91,6 @@ def check_redhat_licenses(nodes=nil)
         report_error(@user_does_not_have_licenses)
       elsif nodes && licenses_count < nodes.count
         report_success(format(@msg_not_enough_licenses, licenses_count))
-      else
-        report_success(@check_licenses_success_msg)
```
Comment (enchantner) [inline lib/astute/redhat_checker.rb:98]: I guess you still need to report success, only without message. But maybe I'm wrong.

Comment (yoojinl) [inline lib/astute/redhat_checker.rb:98]: Yes, you are right.

**Comment by mihgen** [comment]
-1 unless it properly fixed

**Comment by yoojinl** [comment]
I'll fix it.

**Pull request closed without merging**